portal/src/texts/universell-utforming/Roller.mdx

```diff
@@ -0,0 +1,73 @@
+
+Det forventes:
+- Generell forståelse av hva universell utforming er
+- Forståelse av hva som forventes av de ulike rollene
+- Bidra til å sette fokus på universell utforming
+- Hjelpe hverandre til å øke forstålse og kunnskap
+
+## UX
+Designere og UXere er "førsteforsvaret" for universell utforming. Det er helt kritisk for å kunne levere en god opplevelse for brukerene vår at universell utforming er med fra første stund, det kan ikke være noe man legger på etter at man har designet løsningen. 
+
+Det forventes:
+- Forståelse av god informasjonsarktektur (kanskje vi burde lenke til noe i alle disse for å la folk fylle hull feks)
+- Forståelse av god fargebruk og kontrast
+- Forståelse av bevegelse og animasjon
+- Forståelse av hvordan designet blir oversatt til funksjonelle løsninger
+- Kunnskap om hvem våre brukere er og hvordan de bruker våre løsninger
+- Eksperter på brukertesting
+
+## Utvikling
+Utviklerne er ansvarlige for å oversette design og ideer til virkelighet, det innebærer å ha forståelese av hvordan designet blir i den virkelige verden på ulike skjermer, ulike dingser osv. Hvis man jobber med frontend i en eller annen form må man ta et stort ansvar for at våre løsninger skal være tilgjengelig for flest mulig. Man kan ikke gjemme seg bak "backend" eller "fullstack" merkelapp, jobber du med flater vi treffer kunder i, så er dette ditt ansvar.
+
+Det forventes:
+- Forståelse av semantisk HTML
+- Kunnskap om bruk av skjermleser og tastaturnavigasjon
+- Kunnskap om brukerpreferanser som redusert bevegelse
+- Kunnskap om responsiv CSS
+- Kunnskap om avanserte og/eller programatiske verifikasjonsverktøy
+- Automatisering av de deler av prosessen som kan automatiseres
+- Bry seg om brukeropplevelsen i sluttproduktet
+
+## Test
+Testerne er de som i størst grad representerer brukerne, derfor er det viktig at testerne evner å sette seg inn i en rekke forskjellige situasjoner og brukermønstre. Test er sikkerhetsnettet, men det teamet som er ansvarlig for kvaliteten på universell utforming, ikke test alene.
+
+Det forventes:
+- Eksperter på bruk, og hvordan våre kunder bruker, en rekke støtteverktøy, skjermleser, zoom, tastaturnavigasjon m.m.
+- Kritisk blikk på informasjonsarktektur, fargebruk og bevegelse
+- Kunnskap om bruk av verifikasjonsverktøy som Wave eller accessibilityinsights
+- Sette fokus på universell utforming igjennom daglig testing, regresjonstesting ved release og felles testfester
+
+## Produkt
+Som leder for et team eller produkt står man ansvarlig for produktet som leveres, det innebærer at man må ha et forhold til universell utforming i sitt produkt og i sitt team.
```

Comment:
"Som leder for et team eller produkt står man ansvarlig for produktet som leveres", <-- punktum her istedenfor komma